L1Trigger/Configuration/python/ValL1Emulator_cff.py

```diff
@@ -122,11 +125,11 @@
 import L1Trigger.GlobalMuonTrigger.gmtDigis_cfi
 valGmtDigis = L1Trigger.GlobalMuonTrigger.gmtDigis_cfi.gmtDigis.clone()
 #
-valGmtDigis.DTCandidates = cms.InputTag('gtDigis','DT')
-valGmtDigis.CSCCandidates = cms.InputTag('gtDigis','CSC')
-valGmtDigis.RPCbCandidates = cms.InputTag('gtDigis','RPCb')
-valGmtDigis.RPCfCandidates = cms.InputTag('gtDigis','RPCf')
-valGmtDigis.MipIsoData = 'gctDigis'
+valGmtDigis.DTCandidates = cms.InputTag('l1GtUnpack','DT')
+valGmtDigis.CSCCandidates = cms.InputTag('l1GtUnpack','CSC')
+valGmtDigis.RPCbCandidates = cms.InputTag('l1GtUnpack','RPCb')
+valGmtDigis.RPCfCandidates = cms.InputTag('l1GtUnpack','RPCf')
+valGmtDigis.MipIsoData = 'l1GtUnpack'
```

Comment:
@slava77 both are correct. currently we are using GCT (gctDigis) to read out calo regions(RCT). But in MC, we can do both gctDigis and caloStage1Digis. There is no difference.